examples/cmake/modules/gluecodium/gluecodium/runGenerate.cmake

```diff
@@ -31,16 +31,10 @@ function(_generate)
         unset(_build_local_gluecodium)
     endif()
 
-    if(DEFINED ENV{GLUECODIUM_TIME_LOGGING})
-        set(_gluecodium_time time -o "${APIGEN_BUILD_OUTPUT_DIR}/gluecodium-time.log")
-    else()
-        unset(_gluecodium_time)
-    endif()
```

Comment:
This is external (i.e. build system) time measuring using the command line tool "time", please keep it.

Reply:
Done